### DISCUSSION TITLE
changing pwm frequency

### DISCUSSION THREAD

**Discussion by sonyabraham22** [Arduino-IRremote/Arduino-IRremote#957] [Q&A] [answered]
is there anyway to change all pwm pin frequencies to 31372.55 Hz

**Comment by AnalysIR** [comment]
Depending on the platform...you would be lucky to get an exact match to any specific frequency, unless it was an available divisor of the crystal clock.

I note your freq is /510 of 16mHz. 
usually you can set the hardware timers to a target frequency (delve into the Library source code), 

Using sendRAW you should be able to set the carrier close with  hardware PWM configured

**Reply by sonyabraham22** [reply]
thankyou.

**Comment by ArminJo** [comment, marked as answer]
You know the function `void timerConfigForSend(uint8_t aFrequencyKHz)`?

**Reply by sonyabraham22** [reply]
was not aware of that, i am just a beginner in this..thankyou.
